### PR TITLE
Changes to LARC algorightm

### PR DESCRIPTION
**Pull request by Kipok**
I made the following changes to the LARC algorithm (as we discussed in the latest meeting):

  1. Instead of checking for weight/grad norm being close to zero and switching to SGD, we now have min_update parameter that is a minimum value our LARC grad norm update can be.
  2. I removed division by learning rate from the update. This way algorithm becomes more aligned with the usual grad clipping. With this change it is necessary to change parameters accordingly (basically, divide the previous good larc_nu by learning rate, which I did for all our configs). It will be exactly the same for fixed learning rate and should (hopefully) be similar enough for decreasing learning rate. In my preliminary test experiments the new defaults seems to work as well as the old version.
  3. I added more parameters: epsilon (which is what added to grad_norm in the denominator so that we don't need to check for it being zero) and min_update (which is the minimal value that the gradient can be multiplied with). I set some defaults for these values which seem to be reasonable to me, but might not work in all cases (e.g. in float-16, did not check that yet), so I also included them in the parameters. To make all this parameter management easier I combined them all in the larc_params dictionary.

### CODE REVIEW

File: open_seq2seq/models/model.py [hunk 1]
```diff
@@ -338,8 +337,7 @@ def compile(self, force_var_reuse=False):
         summaries=self.params.get('summaries', None),
         colocate_gradients_with_ops=True,
         increment_global_step=True,
-        LARC_nu=self.params.get('larc_nu', None),
-        LARC_mode=self.params.get('larc_mode', 'clip'),
```
Comment (okuchaiev) [inline open_seq2seq/models/model.py]: So LARS is gone now?

Comment (okuchaiev) [reply open_seq2seq/models/model.py]: Found it - now in params, ignore this comment

File: open_seq2seq/optimizers/optimizers.py [hunk 2]
```diff
@@ -376,45 +375,35 @@ class should be sub-class of `tf.Optimizer` that implements
...
         g_norm = tf.norm(tensor=tf.cast(g, tf.float32), ord=2)
 
-        zero_const = tf.constant(0.0, dtype=tf.float32)
-        # this condition is necessary for two reasons.
-        # First, g_norm can be zero. In that case we can not use the usual
-        # formula, but it does not matter what larc_local_lr is, since it will
-        # be multiplied by g = 0.
-        # Second, v_norm can be zero. In that case, we want to switch to the
-        # usual gradient descent since the usual LARC update will always be 0.
-        # Thus, we make larc_local_lr = 1e-5 so that
```
Comment (okuchaiev) [inline open_seq2seq/optimizers/optimizers.py]: Lets also make sure that we have an option to track both updated and "raw" gradients in TB

Comment (Kipok) [reply open_seq2seq/optimizers/optimizers.py]: Added larc_summaries parameter with latest commits

**Pull request merged**